app/apis/bookmarks.js

```diff
@@ -23,9 +23,12 @@ const getBookmarks = async (req, res) => {
                 [req.body.uid, req.body.pid]
             )
         } else {
+            const page = req.body.page == undefined ? 0 : req.body.page;
+            console.log(page)
```

Comment:
oops!

Reply:
make sure you know that when you are querying that you are querying page - 1.

ie. page 1 corresponds to the second page

Reply:
the frontend starts count at 0 so I think it should be fine